util/src/validate.ts

```diff
@@ -63,20 +65,21 @@ class Statistics {
 	}
 }
 
-export function validate(toolOutput: LSIF.Element[], ids: string[], protocolPath: string): number {
-	if (fse.pathExistsSync(protocolPath)) {
+export function validate(toolOutput: LSIF.Element[], ids: string[]): number {
+	const nodeModulesPath: string | null = findNodeModulesPath();
+	if (nodeModulesPath !== null) {
 		readInput(toolOutput);
 
 		checkAllVisited();
 
 		checkVertices(toolOutput.filter((e: LSIF.Element) => e.type === 'vertex')
 								.map((e: LSIF.Element) => e.id.toString()),
-								protocolPath);
+								nodeModulesPath);
 		checkEdges(	toolOutput.filter((e: LSIF.Element) => e.type === 'edge')
 								.map((e: LSIF.Element) => e.id.toString()),
-								protocolPath);
+								nodeModulesPath);
 	} else {
-		console.error(`Error: ${protocolPath} was not found`);
+		console.error(`Error: protocol.d.ts was not found`);
```

Comment:
Would be nice to include the path that was looked in this error, although in that case you might have to log it form findNodeModulesPath